src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
 body {
+  padding: 0;
   margin: 0;
```

Comment:
The 'margin' property is already set to 0 in the universal selector '*' on line 4, so setting it again here for 'body' is redundant.